aqua/qt6/Portfile

```diff
@@ -137,12 +137,12 @@ array set modules {
             d9924d6fd4fa5f8e24458c87f73ef3dfc1e7c9b877a5407c040d89e6736e2634
             46541252
         }
-        "port:ninja port:pkgconfig"
-        "port:assimp port:brotli path:bin/cmake:cmake path:bin/dbus-daemon:dbus port:double-conversion port:freetype
+        "path:bin/cmake:cmake port:ninja port:pkgconfig"
+        "port:brotli path:bin/dbus-daemon:dbus port:double-conversion port:freetype
             path:lib/pkgconfig/glib-2.0.pc:glib2 path:lib/pkgconfig/harfbuzz.pc:harfbuzz
-            port:hunspell path:lib/pkgconfig/icu-uc.pc:icu port:jasper path:include/turbojpeg.h:libjpeg-turbo port:libb2
-            port:libiconv port:libpng port:md4c path:bin/node:nodejs16 path:lib/pkgconfig/libpcre2-posix.pc:pcre2
```

Comment:
removing nodejs16 because that limits your ability to build universal isnt't a good argument. Back in the says it was used so please provide some evidence that it isn't actually needed anywhere anymore.

Reply:
Is "Qt's website indicates that it's needed for Qt WebEngine and Qt PDF, but neither of those are included in this port at the moment." not good enough evidence?  How am I supposed to find evidence of a requirement that doesn't (and from what I can tell, never did) exist?

Reply:
the documentation recommends to install nodejs12, so maybe one solution would be to change this dependency to nodejs12 that is provided as a universal package, until nodejs16 universal package is available ? 

https://doc.qt.io/qt-6/macos-building.html

Reply:
That page also indicates requirements for html5lib, bison, flex, and gperf.  Unsurprisingly, the subset of qt6 we build here doesn't require it because we don't build Qt WebEngine or Qt PDF.  I assume nodejs16 is a leftover from either something else or a time when we did build one of those two modules.  Either way, if you really wanted to follow that, you'd want to add all those dependencies, but IMO adding things because they're in a list of dependencies for modules we don't build is pretty pointless.